src/app/modules/core-bcp/components/review-container/review-container.component.html

```diff
@@ -1,56 +1,69 @@
 <!--#region Section  -->
 
-<!--Section - Heading-->
-<div *ngIf="!hideReviewSection"
-     class="row review--header border-bottom">
-  <div class="col-md-10 {{displayPrintView? 'pl-0 ml-0' : '' }}">
-    <h2>{{header}}</h2>
-  </div>
-  <div *ngIf="!displayPrintView" class="col-md-2 review--icon">
-    <button class="btn btn-edit" (click)="redirect(redirectPath)" aria-label="Edit">
-      Edit <i class="fa fa-pencil fa-lg pl-2" aria-hidden="true"></i>
-    </button>
+<div class='review-container'>
+  <!--Section - Heading-->
+  <div *ngIf="!hideReviewSection"
+      class="row review--header border-bottom">
+    <div class="col-md-10 pl-0 ml-0">
+      <h2>{{header}}</h2>
+    </div>
+    <div *ngIf="!displayPrintView" class="col-md-2 review--icon">
+      <button class="btn btn-edit" (click)="redirect(redirectPath)" aria-label="Edit">
+        Edit <i class="fa fa-pencil fa-lg pl-2" aria-hidden="true"></i>
+      </button>
+    </div>
   </div>
-</div>
 
-<!--Section - Content -->
-<div *ngIf="!hideReviewSection && !showCheckBoxList"
-      class="col-md-12 ml-0 mt-0 {{displayPrintView? 'pl-2 pt-2 pb-2': 'pl-4 pt-4 pb-4 review--background-color'}}">
-  <div *ngFor="let form of sectionItems; let formIndex=index" >
-    <div class="row mb-2" *ngFor="let item of form; let i=index">
-      <ng-container *ngIf='item.label'>
-        <div class='col-md-6'>
-          {{item.label}}{{item.hideColon ? '' : ':'}}
-        </div>
-        <div class='col-md-6'>
-          <b>{{item.value}}</b>
-        </div>
-      </ng-container>
-      <ng-container *ngIf='!item.label'>
-        <div class='col-md-12'>
-          {{item.value}}
-        </div>
-      </ng-container>
+  <!--Section - Content -->
+  <div *ngIf="!hideReviewSection && !showCheckBoxList"
+        class="col-md-12 ml-0 mt-0 {{displayPrintView? 'pl-2 pt-2 pb-2': 'pl-4 pt-4 pb-4 review--background-color'}}">
+    <div *ngFor="let form of sectionItems; let formIndex=index" >
+      <div class="row mb-2" *ngFor="let item of form; let i=index">
+        <ng-container *ngIf='item.label'>
```

Comment:
Suggestion

<div *ngIf='item.label; else NoLabel'>
          <div class='col-md-6'>
            {{item.label}}{{item.hideColon ? '' : ':'}}
          </div>
          <div class='col-md-6'>
            <ng-container *ngIf='containsMultiLine(item.value)'>
              <b [innerHTML]='getMultiLineHTML(item.value)'></b>
            </ng-container>
            <ng-container *ngIf='!containsMultiLine(item.value)'>
              <b>{{item.value}}</b>
            </ng-container>

          </div>
        </div>

<ng-template #NoLabel>
-- Put logic for no label here --
</ng-template>